transport/bdp_estimator.go

```diff
@@ -89,13 +106,14 @@ func (b *bdpEstimator) calculate(d [8]byte) {
 	// If the current sample (which is smaller than or equal to the 1.5 times the real BDP) is
 	// greater than or equal to 2/3rd our perceived bdp AND this is the maximum bandwidth seen so far, we
 	// should update our perception of the network BDP.
-	if float64(b.sample) >= float64(0.66)*float64(b.bdp) && bwCurrent == b.bwMax {
+	if float64(b.sample) >= float64(beta)*float64(b.bdp) && bwCurrent == b.bwMax && b.bdp != bdpLimit {
 		// Put our bdp to be smaller than or equal to twice the real BDP.
 		// We really should multiply with 4/3, however to round things out
```

Comment:
Move this comment to where gamma is defined.  Maybe add something like:

"If gamma were 4/3, we would set our new BDP estimate to double the measured BDP".

Alternatively, express gamma as that scaling factor and have another const of 1.5 for the 1.5 round trips covered by the sample.  Or even `gamma = 3/1.5`?